Documentation/tutorial/docfx.exe_user_manual.md

```diff
@@ -296,6 +297,37 @@ _disableAffix         | bool    | Indicate whether to show the affix bar on the
 _disableContribution  | bool    | Indicate whether to show the `View Source` and `Improve this Doc` buttons.
 _gitContribute        | object  | Customize the `Improve this Doc` URL button for public contributors. Use `repo` to specify the contribution repository URL. Use `branch` to specify the contribution branch. Use `path` to specify the folder for new overwrite files .
 
+#### 3.2.3 Separated metadata files for global metadata and file metadata
+
+Take global metadata for example, you can set `globaMetadataFiles` in `docfx.json` or `--globalMetdataFiles` in build command line. The usage of `fileMetadataFiles` is the same as `globalMetadataFiles`.
+
+There're some examples:
+
++ use `globalMetadataFiles` in `docfx.json`
+    ```json
+    ...
+    "globalMetadataFiles": ["global1.json", "global2.json"],
```

Comment:
file metadata is not that simple, we should give some samples here.

---
In reply to: [74208263](https://github.com/dotnet/docfx/pull/591#discussion_r74208263) [](ancestors = 74208263)